tests/strategies/test_fed_avg.py

```diff
@@ -78,15 +78,14 @@ class MyAlgo(dummy_algo_class):
         @remote_data
         def train(
             self,
-            x: np.ndarray,
-            y: np.ndarray,
+            datasamples: np.ndarray,
             shared_state,
         ):
             if shared_state is not None:
                 # We predict the shared state, an array of 0.5
                 assert int((shared_state.avg_parameters_update == np.ones(1) * 0.5).all())
 
-            return FedAvgSharedState(n_samples=len(x), parameters_update=[np.asarray(e) for e in x])
+            return FedAvgSharedState(n_samples=len(datasamples), parameters_update=[np.asarray(e) for e in datasamples])
```

Comment:
```suggestion
            return FedAvgSharedState(n_samples=len(datasamples), parameters_update=[np.asarray(x) for x,_ in datasamples])
```

not sure about this one, if it is important to return an array of x or x,y

Reply:
The test is a bit flaky has dataset has the same size on org1 and 2, this is why it works. I updated it